config/ros2-test.conf

```diff
@@ -0,0 +1,23 @@
+#
+# ROS Test Setting
+#
+# This is an test configuration file for for rcl_logging_syslog.
+#
+# For more information install rsyslog-doc and see
+# /usr/share/doc/rsyslog-doc/html/configuration/index.html
+#
+
+# Test Message Format Template
+# See more details for https://www.rsyslog.com/doc/configuration/properties.html
+#$template TestFormat,"%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\n"
+$template TestFormat,"%msg:2:$%\n"
+
+# Directory and File Name Test Template
+# See more details for https://www.rsyslog.com/doc/configuration/properties.html
+### e.g) /var/log/rcl_logging_syslog/test_logging_interface_1405655_1727291103.log
+$template TestFileName,"/var/log/rcl_logging_syslog/%programname%_%procid%_%$now-unixtimestamp%.log"
+
+# Log locally with specified templated file name.
+local1.* ?TestFileName;TestFormat
+local2.* ?TestFileName;TestFormat
+local3.* ?TestFileName;TestFormat
```

Comment:
even user system uses those facilities already, it just work out of the box. rsyslog scan the all configuration and output the logging data to the specified sink.